app/src/main/java/com/ciphereck/viralinfectiontracker/reports/ReportsNavigation.java

```diff
@@ -0,0 +1,56 @@
+package com.ciphereck.viralinfectiontracker.reports;
+
+import androidx.annotation.NonNull;
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.fragment.app.Fragment;
+import androidx.navigation.Navigation;
+import androidx.navigation.ui.NavigationUI;
+
+import com.ciphereck.viralinfectiontracker.R;
+import com.google.android.material.bottomnavigation.BottomNavigationView;
+
+import android.os.Bundle;
+import android.view.MenuItem;
+
+public class ReportsNavigation extends AppCompatActivity {
+
+    private BottomNavigationView navView;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_reports_navigation);
+
+        navView = findViewById(R.id.nav_view);
+
+        navView.setItemIconTintList(null);
+        navView.setOnNavigationItemSelectedListener(navListener);
```

Comment:
use Rx instead of navigation listener for cleaner code
Check Jake Wharton Rx or RxBinding

Convert Observable from events and subscribe to it when the view gets initialized